package.json

```diff
@@ -1211,7 +1212,6 @@
     "@types/chai": "^4.3.17",
     "@types/debug": "^4.1.12",
     "@types/glob": "^7.2.0",
-    "@types/jest": "^26.0.24",
```

Comment:
I couldn't find any references to jest in the codebase, so figured we don't need this.